nsxt/provider.go

```diff
@@ -1168,6 +1169,11 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
+	connector := getPolicyConnector(clients)
+	clients.DefaultSpanPath, err = getDefaultSpan(connector)
```

Comment:
@aruntony005 this should only run for v9.1 and above - we have no spans before this.
This is why you get that notfound error (that API doesn't exist in 9.0)